pagopa/io-app
# chore(Carta della cultura): [AP-5] Add cdc bonus entry points

## Short description
This PR adds the bonus list and the carousel entry points for the CDC bonus.

Since that the when the bonus item is pressed the user should navigate to the CDC service details, if the service is not loaded when the components are mounted, the action to load the service is dispatched.

## List of changes proposed in this pull request
- Added the CDC bonus to the `AvailableBonusScreen`
- Added the CDC bonus to the `FeaturedCardCarousel`

https://user-images.githubusercontent.com/11773070/168118526-094e0974-dc63-4b70-b719-4206f2319018.mp4



## How to test
To test the functionality check out the branch on [this PR](https://github.com/pagopa/io-dev-api-server/pull/171) of the dev-server.
Remember to set the local and the remote feature flag to true, and play with the configuration of the bonus.
